test/WebJobs.Script.Tests/Rpc/RpcMessageConversionExtensionsTests.cs

```diff
@@ -247,5 +255,49 @@ internal static ClaimsIdentity MockEasyAuth(string provider, string name, string
 
             return identity;
         }
+
+        [Theory]
+        [InlineData("application/octet-stream", true)]
+        [InlineData("image/png", true)]
+        [InlineData("application/octet-stream", false)]
+        [InlineData("image/png", false)]
+        public void HttpObjects_RawBodyBytes_Image_Length(string contentType, bool rawBytesEnabled)
+        {
+            var logger = MockNullLoggerFactory.CreateLogger();
+            var capabilities = new Capabilities(logger);
+            capabilities.UpdateCapabilities(new MapField<string, string>
+            {
+                { LanguageWorkerConstants.RawHttpBodyBytes, rawBytesEnabled.ToString() }
+            });
+
+            FileStream image = new FileStream(TestImageLocation, FileMode.Open, FileAccess.Read);
+            byte[] imageBytes = FileStreamToBytes(image);
+            long imageLength;
+
+            imageLength = imageBytes.Length;
+
+            var headers = new HeaderDictionary();
+            headers.Add("content-type", contentType);
+            HttpRequest request = HttpTestHelpers.CreateHttpRequest("GET", "http://localhost/api/httptrigger-scenarios", headers, imageBytes);
+
+            var rpcRequestObject = request.ToRpc(logger, capabilities);
+            if (rawBytesEnabled)
+            {
+                Assert.Equal(imageLength, rpcRequestObject.Http.RawBody.Bytes.ToByteArray().Length);
+            }
+            else
+            {
+                Assert.NotEqual(imageLength, rpcRequestObject.Http.RawBody.Bytes.ToByteArray().Length);
```

Comment:
Isn't this a string?

Reply:
Yes, you're right. Added a test to check that the string length is equal to what we expect from the ToRpc function.